src/rpc/services/initHandshake/InitHandshakeRpcMethods.ts

```diff
@@ -21,7 +20,7 @@ class InitHandshakeRpcMethods extends ARpcMethods {
             this.p2pManager.stateManager.timeConfig.agreementTime
         ) {
             this.p2pManager.disconnectConnection(this.senderTransport);
-            console.log(
+            this.service.logger.debug(
```

Comment:
here and eveywhere
do not use <>.<>.<>.logger
logger should be scoped to its own class

just `this.logger`